pennylane/debugging/debugger.py

```diff
@@ -43,7 +43,7 @@ class PLDB(pdb.Pdb):
     def __init__(self, *args, **kwargs):
         """Initialize the debugger, and set custom prompt string."""
         super().__init__(*args, **kwargs)
-        self.prompt = "[pldb]: "
+        self.prompt = "[pldb] "
```

Comment:
I see why you updated the docstrings. This is definitely more consistent with `pdb` so I'm on board with this. cc-ing @trbromley just in case.